README.md

```diff
@@ -33,6 +33,8 @@ Usage
 | ------ | -------- |------------ |
 | `consul`    | _(required)_ | The location of the Consul instance to query (may be an IP address or FQDN) with port. |
 | `template`  | _(required)_ | The input template, output path, and optional command separated by a colon (`:`). This option is additive and may be specified multiple times for multiple templates. |
+| `https`     | | Use HTTPS while talking to Consul. Requires the Consul server to be configured to serve secure connections.
+| `insecure`  | | Ignore certificate warnings. Only used if `https` is enabled.
```

Comment:
I think we should name this something more specific like "--no-ssl-verify" or something. "Insecure" feels sad to me. What do you think? Any other ideas?